src/frontend/magnify/src/components/rooms/RoomOverview/RoomOverview.tsx

```diff
@@ -0,0 +1,83 @@
+import { defineMessages, useIntl } from 'react-intl';
+import React from 'react';
+import { Box, Button, Card } from 'grommet';
+import { useController } from '../../../controller';
+import { useQuery } from 'react-query';
+import { LoadingButton, RowsList, SettingsSVG } from '../../design-system';
+import MeetingRow from '../../meetings/MeetingRow/MeetingRow';
+import { GroupRow } from '../../groups';
+
+const messages = defineMessages({
+  meetingListTitle: {
+    id: 'components.rooms.roomOverview.meetingListTitle',
+    defaultMessage: '{numberOfRows} Meetings in this room',
```

Comment:
I don't know if you know it but you can format you message with plural rules https://formatjs.io/docs/core-concepts/icu-syntax